modules/cnv-creating-local-block-pv.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-importing-virtual-machine-images-datavolumes-block.adoc
+// * cnv/cnv_users_guide/cnv-cloning-vm-disk-into-new-datavolume-block.adoc
+// * cnv/cnv_users_guide/cnv-uploading-local-disk-images-block.adoc
+
+[id="cnv-creating-local-block-pv_{context}"]
+= Creating a local block PersistentVolume
+
+Create a local block PersistentVolume (PV) on a node by populating a file and 
+mounting it as a loop device. This loop device can then be referenced in a 
+PV configuration as a `Block` volume and used as a block device for a 
+virtual machine image. 
+
+.Procedure
+
+. Log in as `root` to the node on which to create the local PV. This procedure 
+uses `node01` for its examples.
+
+. Create a file and populate it with null characters. The following example creates a file
+`loop10` with a size of 2Gb.
++
+----
+$ dd if=/dev/zero of=<loop10> bs=100M count=20
```

Comment:
Loop signifies that it's a loop device, and 10 is just an integer. Would it be better if it was loop1 ?
I believe it can be called anything, but if it's called loop then whatever filesystem admin that comes across it will know what it is